buildSrc/src/main/java/com/google/android/material/composethemeadapter/dependencies.kt

```diff
@@ -25,10 +25,10 @@ object Libs {
 
     const val gradleMavenPublishPlugin = "com.vanniktech:gradle-maven-publish-plugin:0.12.0"
 
-    const val mdc = "com.google.android.material:material:1.1.0"
+    const val mdc = "com.google.android.material:material:1.2.1"
 
     object Kotlin {
-        const val version = "1.4.0"
+        const val version = "1.4.10"
```

Comment:
Not sure on this one. This will force all consumers to use 1.4.10 which probably isn't what we want